### PR TITLE
separate supplemental resources from material section

### LINKED ISSUE #570: separate supplemental resources from required materials. (multiple assignees encouraged)
On one side, some apprentices think that they need to throughly look through every material, and are getting way behind because of it.  On the other, many well-meaning volunteers have thrown the kitchen sink into the "materials" section, when many or even all of them should be considered "supplemental resources" (supplemental meaning additional, extra, not required). 

Look through topic outlines, and change one so that:
- the "slides" are part of the "materials" section
- the "lesson" explicitly directs how to use the mandatory "materials" (ex: "Read through [these slides]() )
- optional materials are under a "supplemental resources" section.
- if you finish, or notice a topic outline looks fine, please check it off on this list or let a Techtonica admin know it's complete.

Topic outlines whose materials sections not been reviewed:
- [x] https://github.com/Techtonica/curriculum/blob/master/professionalism/professionalism.md
- [x] https://github.com/Techtonica/curriculum/blob/master/ergonomics/ergonomics.md

- [x] https://github.com/Techtonica/curriculum/blob/master/command-line/vim.md
 
- [x] https://github.com/Techtonica/curriculum/blob/master/html/html.md

- [x] https://github.com/Techtonica/curriculum/blob/master/git-version-control/gitignore.md
 
- [x] https://github.com/Techtonica/curriculum/blob/master/css/css.md

- [x] https://github.com/Techtonica/curriculum/blob/master/css/media-queries.md
 
- [x] https://github.com/Techtonica/curriculum/blob/master/ui-ux-design/ui-ux-design.md
- [x] https://github.com/Techtonica/curriculum/blob/master/web-patterns/web-patterns.md
- [x] https://github.com/Techtonica/curriculum/blob/master/bootstrap/bootstrap.md
- [x] https://github.com/Techtonica/curriculum/blob/master/professionalism/building-confidence.md

- [x] https://github.com/Techtonica/curriculum/blob/master/github-collaboration/github-collaboration.md
- [x] https://github.com/Techtonica/curriculum/blob/master/reading-and-writing-documentation/documentation.md
- [x] https://github.com/Techtonica/curriculum/blob/master/deploying/deploying-1-personal-site.md
 
- [x] https://github.com/Techtonica/curriculum/blob/master/javascript/javascript-1.md
- [x] https://github.com/Techtonica/curriculum/blob/master/javascript/javascript-2.md

- [x] https://github.com/Techtonica/curriculum/blob/master/runtime-complexity/runtime-complexity.md
- [x] https://github.com/Techtonica/curriculum/blob/master/testing-and-tdd/jasmine-testing.md
- [x] https://github.com/Techtonica/curriculum/blob/master/debugging/debugging.md
- [x] https://github.com/Techtonica/curriculum/blob/master/databases/relational-databases.md
- [x] https://github.com/Techtonica/curriculum/blob/master/react-js/react.md
- [x] https://github.com/Techtonica/curriculum/blob/master/data-structures/intro-to-data-structures.md
- [x] https://github.com/Techtonica/curriculum/blob/master/data-structures/stack.md
- [x] https://github.com/Techtonica/curriculum/blob/master/data-structures/deque.md

- [x] https://github.com/Techtonica/curriculum/blob/master/data-structures/trees.md
 
- [x] https://github.com/Techtonica/curriculum/blob/master/algorithms/intro-to-algorithms.md
- [x] https://github.com/Techtonica/curriculum/blob/master/algorithms/searching.md
- [x] https://github.com/Techtonica/curriculum/blob/master/algorithms/sorting.md

- [x] https://github.com/Techtonica/curriculum/blob/master/open-source/open-source.md
 
- [x] https://github.com/Techtonica/curriculum/blob/master/roles-in-tech/machine-learning.md

### PR DESCRIPTION
**Pull request by vsharma20v**
separate supplemental resources from the material section in chrome-developer-tools.md file and fixed some typo errors
Issue Fixed #570

### CODE REVIEW

File: chrome-developer-tools/chrome-developer-tools.md [hunk 1]
```diff
@@ -27,22 +27,24 @@ Chrome Developer Tools help tremendously with front-end development (HTML, CSS &
 - How to access the JavaScript console to run and debug code
 - Participants will learn the various keyboard shortcuts to open Chrome Developer Tools
 
-### Supplemental Materials
+### Materials
+- [Chrome Developer Tools (video walkthrough of slides)](https://drive.google.com/file/d/1LV6lOI0MRKUVSaDlSipiy6JX8EtEaYjH/view?usp=sharing)
+- [Chrome Developer Tools (slides)](https://docs.google.com/presentation/d/1z8aIzOxV5L-zW3MZmZhX67DgyLq_4ErO4yTS4CIp7R0/edit?usp=sharing)
```
Comment (Kundan28) [inline chrome-developer-tools/chrome-developer-tools.md]: @vsharma20v Please make sure the presentation follows all the formatting guidelines mentioned in #600 . As I am not having the edit access of the slideshow so I can't review it.

File: chrome-developer-tools/chrome-developer-tools.md [hunk 2]
```diff
@@ -27,22 +27,24 @@ Chrome Developer Tools help tremendously with front-end development (HTML, CSS &
...
 
+### Supplemental Materials
 - [Quick Source Viewer Chrome Extension](https://chrome.google.com/webstore/detail/quick-source-viewer/cfmcghennfbpmhemnnfjhkdmnbidpanb?hl=en-US)
 
 ### Lesson
 - [Chrome Developer Tools (video walkthrough of slides)](https://drive.google.com/file/d/1LV6lOI0MRKUVSaDlSipiy6JX8EtEaYjH/view?usp=sharing)
-
 - [Chrome Developer Tools (slides)](https://docs.google.com/presentation/d/1z8aIzOxV5L-zW3MZmZhX67DgyLq_4ErO4yTS4CIp7R0/edit?usp=sharing)
```
Comment (Kundan28) [inline chrome-developer-tools/chrome-developer-tools.md]: The two things that are in the **Lesson** section are already mentioned in the **Material** section. So I think Lesson section could be removed.

Comment (vsharma20v) [reply chrome-developer-tools/chrome-developer-tools.md]: @alodahl @Kundan28 
what about naming material section as Lesson also?
i.e. material / lesson

Comment (Kundan28) [reply chrome-developer-tools/chrome-developer-tools.md]: No, I don't think it would look good.

Comment (vsharma20v) [reply chrome-developer-tools/chrome-developer-tools.md]: Done, I made the request changes expect the slideshow guidelines once.

Comment (Kundan28) [reply chrome-developer-tools/chrome-developer-tools.md]: I can't review the Slideshow as I don't have edit access. But rest of the thing is good. And this PR is ready to be merged.

Comment (Kundan28) [reply chrome-developer-tools/chrome-developer-tools.md]: But the slideshow looks good to me.

**Pull request merged**